stubs/CoreGenericFunctions.phpstub

```diff
@@ -1224,7 +1224,7 @@ function preg_replace_callback($pattern, $callback, $subject, int $limit = -1, &
  *
  * @param non-empty-string $pattern
  * @param string $subject
- * @param mixed $matches
+ * @param array|null $matches
```

Comment:
Looks like you accidentally committed more changes than intended.

Reply:
I'm not sure we should. It looks like non-array vars are accepted just fine: https://3v4l.org/7pgBM.